test/integration-tests/IntegrationTests.SqlClient/SqlClientCollection.cs

```diff
@@ -33,7 +34,6 @@ public class SqlClientFixture : IAsyncLifetime
     {
         private const int DatabasePort = 1433;
         private const string DatabaseImage = "mcr.microsoft.com/mssql/server:2019-CU15-ubuntu-20.04";
-        private const string DatabasePassword = "@someThingComplicated1234";
```

Comment:
Isn't this password still hardcoded in one of the yml files? Perhaps one of the other changes in this PR is overriding that setting, but it's not clear to me?

Reply:
My VS Code doesn't find `@someThingComplicated1234` in any other place.

Reply:
It looks like I was remembering a change in an [older commit](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/669/commits/80226cdef371e8fc7a00c9bab9f8235baaf0cd38) that was later removed.